index.js

```diff
@@ -115,7 +115,13 @@ class SideMenu extends Component {
    */
   handlePanResponderMove(e: Object, gestureState: Object) {
     if (this.state.left.__getValue() * this.menuPositionMultiplier() >= 0) {
-      this.state.left.setValue(this.prevLeft + gestureState.dx);
+      let newLeft = this.prevLeft + gestureState.dx;
+
+      if (!this.props.bounceBackOnOverdraw && Math.abs(newLeft) > this.props.openMenuOffset) {
```

Comment:
I would like to have an ability to configure it (based on my previous experience with various app I've done). It doesn't hurt us yet it's useful (e.g. I am using `bounceBackOnOverdraw` in my projects - I like the ability to toggle more than it's needed. That's highly app specific UX and should be left up to users w/o extra hacking in my opinion).